src/domains/Townhall/TownHallHistory/TownHallHistory.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem, { ListItemProps } from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Divider from '@material-ui/core/Divider';
+import InboxIcon from '@material-ui/icons/Inbox';
+import DraftsIcon from '@material-ui/icons/Drafts';
+import { ListItemAvatar, Avatar } from '@material-ui/core';
+import { getTownhallList } from '../api';
```

Comment:
Don't forget to remove unnecessary imports if they are not being used.

Reply:
Resolved